src/app/pages/client-reviews/client-reviews.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Subscription } from 'rxjs';
+import { Component, OnInit, Input } from '@angular/core';
+import { ClientReviewsService } from 'src/app/services/client-reviews/client-reviews.service';
+import { ActivatedRoute, Router } from '@angular/router';
+import { APPCONFIG } from 'src/app/config';
+import { ToastrService } from 'ngx-toastr';
+import { NgxSpinnerService } from 'ngx-spinner';
+
+@Component({
+  selector: 'app-client-reviews',
+  templateUrl: './client-reviews.component.html',
+  styleUrls: ['./client-reviews.component.scss']
+})
+export class ClientReviewsComponent implements OnInit {
+  subscription: Subscription = new Subscription();
+  count: number;
+  results: any[];
+  url: string;
+  next: string;
+  prev: string;
+  disabledNext = false;
+  disabledPrev = false;
+  profileImage: string;
+
+  constructor(
+    private reviewsService: ClientReviewsService,
+    private route: ActivatedRoute,
+    private toastrService: ToastrService,
+    private router: Router,
+    private spinner: NgxSpinnerService
+  ) {
+    this.profileImage = 'assets/img/people.png';
+  }
+
+  ngOnInit(): void {
+    this.spinner.show();
+    this.route.paramMap.subscribe(result => {
+      const clientId = result.get('clientId');
+      this.url = `${APPCONFIG.base_url}/auth/${clientId}/reviews/`;
```

Comment:
@dorothyas , what do you think of using the `environment` file directly here so that we have
```
`{environment.base_url}/auth/${clientId}/reviews/`
```